### ISSUE TITLE
Will this work to forward different mining users?

### ISSUE THREAD

**Issue by SecretRecipe** [p4u/stratum-relay#9]
Will this allow me to forward different mining users to a pool and have them be able to use their own wallet addresses etc?

**Comment by p4u** [comment]
Yes, that's what the relay does. See the PDF https://github.com/p4u/stratum-relay/blob/master/stratum_relay.pdf

**Comment by SecretRecipe** [comment]
So far, when I specify the -u, the miner that connects is stuck with that, not the -u from their bat. I can't find a literal guide how to get it to work like you say and "relay".

**Comment by p4u** [comment]
I did not understood your question. Now I think I do but it's hard to see a use cause.

No, multiple destination users for the pool is not supported. See this line in the code https://github.com/p4u/stratum-relay/blob/13af2446cc413bbff0f22b00668df55d53b48bec/strelay.py#L126

The code might by modified to allow multiple users separated by comma, split the string and store users in an array. 

Then modify this line too https://github.com/p4u/stratum-relay/blob/13af2446cc413bbff0f22b00668df55d53b48bec/strelay.py#L143 For using one of the users in the array (and increase array index to let the next connection to use another user).

Does not look like a hard modification to achieve, however I've no miners anymore for testing it and no motivation for working on it anymore (people makes money by using my open source software, but no BTC donations so far) so feel free to do it yourself.

Cheers.
